### PR TITLE
30% Squash.

### PR DESCRIPTION
**Pull request by malcyL**

### CODE REVIEW

**Comment by malcyL** [comment]
Is there a better place to put this than Source Control? I tried linking to it from the source control page and adding nested pages under it in the menu. The nesting of the menu items broke the styling though.

File: topics/source-control.md [hunk 1]
```diff
@@ -19,8 +19,87 @@ Committing often is also a good practice. Every committed revision gives you a r
...
+
+![Picking Commits During Rebase](/images/topics/squash/squash_example_002.png){: .center-image }
+
+In the editor, leave the first commit unchanged, but change the following commits from "pick" to "squash" or just "s" as shown below.
+
+![Selecting Commits to Squash](/images/topics/squash/squash_example_003.png){: .center-image }
+
+Save and exit the file.
+
+You now need to write the commit log message for the single commit being created by the squash. An editor is opened which displays the log messages for each of the squshed commits.
```
Comment (rgubby) [inline topics/source-control.md]: typo: "squshed" -> "squashed"

File: topics/source-control.md [hunk 2]
```diff
@@ -19,8 +19,87 @@ Committing often is also a good practice. Every committed revision gives you a r
...
+
+While working on a branch, commits provide a rollback positions and commit messages reflect this. When merging, the commit message becomes part of the master branches commit log. The content of the commit message at that point can help with things like release tracking and linking or even automatically closing GitHub issues.
+
+For example, using the git command:
+
+{% highlight bash %}
+git log --pretty=format:"%h | %an | %s" ^v1.0.0 v1.1.0
+{% endhighlight %}
+
+shows the commits between tags v1.0.0 and v1.1.0 i.e what is in the release v1.1.0. This can be used to update release notes and the status of stories in Jira etc. It is the single point of truth of what is contained in a release.
```
Comment (rgubby) [inline topics/source-control.md]: capital S for "shows"

Comment (rgubby) [inline topics/source-control.md]: I wouldn't mention Jira by name - rather reference something more general.

**Comment by rgubby** [comment]
Looks good, but you really should squash this pull request! :-)

**Pull request merged**